libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts

```diff
@@ -351,6 +351,34 @@ async function addSettingsBotComponents(services: ServiceCollection, configurati
     }
 }
 
+async function addComposerConfiguration(configuration: Configuration): Promise<void> {
```

Comment:
This can be synchronous, is it intentionally async?

Reply:
Nope - refactored something out that was deemed unnecessary.